includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -723,7 +667,7 @@ public function javascript_params() {
 	 * Load public scripts and styles.
 	 *
 	 * @since   3.1.0
-	 * @version 5.2.0
+	 * @version x.x.x
```

Comment:
`function scripts` is not getting changed now, so we can revert `x.x.x`.

Reply:
I'll update and add also a changelog entry